develop.sh

```diff
@@ -5,63 +5,53 @@ if [[ ! -d "docker" ]]; then
     exit -1
 fi
 
-function invoke_docker_compose_spark {
-	docker-compose -f docker/docker-compose.spark.yml \
-				-p listenbrainzspark \
-				"$@"
-}
-
 function invoke_docker_compose {
-	docker-compose -f docker/docker-compose.yml \
-				-p listenbrainz \
-				"$@"
+    docker-compose -f docker/docker-compose.yml \
```

Comment:
It'd be great if we could remove the whitespace changes in this file.

Reply:
I think indentation looks better now. Not sure if there is a standard we follow.